deployments/bscmainnet/.migrations.json

```diff
@@ -3,5 +3,6 @@
   "vbnbadmin_deploy": 1697477430,
   "xvs_vault_configuration": 1697032901,
   "configure_prime": 1697032901,
-  "configure_xvs_vaults": 1697032901
+  "configure_xvs_vaults": 1697032901,
+  "vai_controller_config": true
```

Comment:
Is `true` enough? Don't we need a block number?

Reply:
Yes it checks for a truthy value - https://github.com/wighawag/hardhat-deploy/blob/e0ffcf9e7dc92b246e832c4d175f9dbd8b6df14d/src/DeploymentsManager.ts#L1186

But we could also put a block number for completeness